Documentation/Crowdin.md

```diff
@@ -0,0 +1,81 @@
+CROWDIN CRASH COURSE
+Link to CrowdIn GitHub integration for this
+project: https://crowdin.com/project/vocable-android/apps/system/github
+
+CrowdIn is a crowd-sourced translation tool for software projects. Vocable uses a CrowdIn file-based
+integration with GitHub to make a PR whenever a new translation is added to CrowdIn. Read the docs:
+https://support.crowdin.com/github-integration/
+
+CrowdIn should automatically raise a PR to the `main` branch whenever translations are updated. The
+sync schedule is currently set to 1 hour, or you have the option to hit the "Sync Now" button on the
+integration page (linked at the top of this file). This PR should involve only string resource
+files.
+
+POTENTIAL PITFALLS
+
+* It is strongly recommended that you do NOT alter the folder structure displayed on the "Sources"
+  tab of the CrowdIn project. This structure was automatically generated when the integration was
+  set up. If the structure of the project no longer matches this structure, or CrowdIn is otherwise
+  not working, you may need to spin up
+  a new GitHub integration.
+* Only the person who created an integration can edit the same integration, and only one integration
+  can be active on a single GitHub repo.
+* To create a new integration, you must have admin privileges on the GitHub repo - you can check
+  this through CrowdIn by navigating to the Integrations tab and clicking "Add Repository". The
+  drop-down will populate after a little while - leave the tab open, it DOES take some time to
+  retrieve your GitHub profile. If you have the correct privileges, you'll
+  see `willowtreeapps/vocable-android` in the repos list, grayed out since you can't create a second
+  integration for the same repo.
+* BEFORE DELETING AN INTEGRATION, download the latest translations from CrowdIn so that they can be
```

Comment:
I'm a little nervous about outlining such specific steps in CrowdIn because: it's verbose, it may become out of date, and I think it hides what we learned/why you'd delete an integration anyway.

I think it would be valuable to include the anecdote that we saw having multiple sources in CrowdIn cause problems with non-english languages being downloaded as English. Unclear if that was a CrowdIn bug or not, but we were able to resolve it by:
* Manually integrating the latest translations
* Deleting all sources and integrations
* Re-adding the integration and allowing it to pull the current translations

Reply:
Sure, I can walk this back a little and include that anecdote instead.